nixos/modules/services/matrix/synapse/default.nix

```diff
@@ -134,6 +134,7 @@ in {
     (mkRemovedOptionModule [ "services" "matrix-synapse" "macaroon_secret_key" ] "Pass this value via extraConfigFiles instead" )
     (mkRemovedOptionModule [ "services" "matrix-synapse" "registration_shared_secret" ] "Pass this value via extraConfigFiles instead" )
 
+    (import ./workers.nix { inherit commonConfigFile format pluginsEnv; })
```

Comment:
Not sure how well this interacts with the module system

Reply:
I was also unsure about this, but I think it's fine